CSharpMath.Tests/PreTypesetting/MathListBuilderTest.cs

```diff
@@ -973,16 +973,16 @@ public class MathListBuilderTest {
 
       Assert.Single(list);
       var latex = MathListBuilder.MathListToString(list);
-      Assert.Equal(RemoveWhiteSpacesFromString(latex), input);
+      Assert.Equal(input, RemoveWhiteSpacesFromString(latex));
     }
 
     [Fact]
-    public void TestListToString() {
+    public void TestListToString_int_a_to_b() {

```

Comment:
`int` could stand for "integer" or "integral", but when someone is looking at the test panel, they might be surprised to find out that "int" actually means "integral" here. Better to give this test a name like `TestListToString_Integral`.